src/ProjectTemplates/BlazorTemplates.Tests/BlazorTemplates.Tests.csproj

```diff
@@ -0,0 +1,162 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- Shared testing infrastructure for running E2E tests using selenium -->
+  <Import Project="$(SharedSourceRoot)E2ETesting\E2ETesting.props" />
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TestGroupName>ProjecteTemplates.E2ETests</TestGroupName>
+    <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
+
+    <RunTemplateTests Condition="'$(RunTemplateTests)' == ''" >true</RunTemplateTests>
+    <SkipTests Condition="'$(RunTemplateTests)' != 'true'">true</SkipTests>
+    <SkipHelixArm>true</SkipHelixArm>
+    
+    <!-- Some tests still do not work on Helix, so continue running these on azdo for now-->
+    <SkipTests Condition="'$(RunTemplateTests)' == 'true'">false</SkipTests>
```

Comment:
Remove this, and replace with `<BuildHelixPayload>false</BuildHelixPayload>` and update comment